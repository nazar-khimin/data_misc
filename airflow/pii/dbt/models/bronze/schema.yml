version: 2

models:
  - name: raw_data
    columns:
      - name: unique_id
        description: System-generated UUID per row
        tests:
          - not_null
          - unique
      - name: accessed_at
        description: Access time, replaced with yesterday's date if today is not some specific day
        tests:
          - not_null
  - name: raw_source