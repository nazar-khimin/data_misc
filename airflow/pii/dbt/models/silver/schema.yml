version: 2

models:
  - name: staging_dim_address
  - name: staging_dim_date
  - name: staging_dim_finance
  - name: staging_dim_person
  - name: staging_fact_network_usage