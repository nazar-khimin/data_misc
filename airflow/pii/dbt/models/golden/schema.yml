version: 2

models:
  - name: non_pii_data
  - name: payment_data
  - name: pii_data
  - name: technical_data